mdn/developer-portal
# 145: Add country filter to events and people

Fixes #145.

This changeset adds support for the filtering of events and people by country.

## Key changes

- Adds support for "country" field on person model.
- Splits the existing "location" field into "city" and "country" for external events.
- Adds the country filter to the sidebar.
- Adds the required meta to event and person cards to allow filtering.

## How to test

- Add countries to one or more events or people.
- View directory pages and filter by a country from the left sidebar.
- Refresh the page to ensure state is maintained within the URL.
